RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationByHPSSelection.cc

```diff
@@ -258,10 +258,13 @@ PFRecoTauDiscriminationByHPSSelection::discriminate(const reco::PFTauRef& tau) c
   }
   // Now check the pizeros
   BOOST_FOREACH(const reco::RecoTauPiZero& cand, tau->signalPiZeroCandidates()) {
+    double dEta = std::max(0., fabs(cand.eta() - tauP4.eta()) - cand.bendCorrEta());
+    double dPhi = std::max(0., reco::deltaPhi(cand.phi(), tauP4.phi()) - cand.bendCorrPhi());
```

Comment:
deltaPhi returns a signed value.
From the definition of bendCorrPhi, I understand now that deltaPhi here is expected to be "std::abs(deltaPhi)"
This looks like a bug